packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Authorization Servers
+meta:
+  - name: description
+    content: An overview of what an authorization server is and the types of authorization servers available at Okta.
+---
+## What is an authorization server
+
+At its core, an authorization server is simply an engine for minting OpenID Connect or OAuth 2.0 tokens. An authorization server is also used to apply access policies. Each authorization server has a unique issuer URI and its own signing key for tokens to keep a proper boundary between security domains.
+
+## What you can use an authorization server for
+
+You can use an authorization server to perform Single Sign-On (SSO) with Okta for your OpenID Connect (OIDC) apps, and to secure your own APIs and provide user authorization to access your web services.
+
+OIDC is used to authenticate users with a web app. The app uses the ID token that is returned from the authorization server to know if a user is authenticated and to obtain profile information about the user, such as their username or locale. OAuth 2.0 is used to authorize user access to an API. An app uses an access token to validate a user's identity. When using OIDC or OAuth, the authorization server authenticates a user and issues an ID token and/or an access token.
+
+> **Note:** You can't mix tokens between different authorization servers. By design, authorization servers don't have trust relationships with each other.
```

Comment:
Is "trust relationship" an industry wide term used to describe authorization server relationships? Is there a more layman term we could use? Essentially here we are saying that the authorization servers don't talk to each other, right?